### PR TITLE
Add _total postfix for counter metrics

### PR DESCRIPTION
**Pull request by clwluvw**
Also there were a bug for `keepalived_packet_len_err` metric that was exported as a Gauge Value

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,28 +23,28 @@ cs                 | Health Cehck script path to be execute for each VIP.
...
+|--------------------------------------|-----------------------------------
+| keepalived_up                        | Status of Keepalived service
+| keepalived_vrrp_state                | State of vrrp
+| keepalived_check_script_status       | Check Script status for each VIP
+| keepalived_garp_delay_total          | Gratuitous ARP delay
+| keepalived_advert_rcvd_total         | Advertisements received
+| keepalived_advert_sent_total         | Advertisements sent
+| keepalived_become_master_total       | Became master
+| keepalived_release_master_total      | Released master
+| keepalived_packet_len_err_total      | Packet length errors
```
Comment (SuperQ) [inline README.md:36]: Another minor naming hint, we try and avoid abbreviations for metric names. It makes it a little more verbose, but easier to read by humans.

```
keepalived_packet_length_errors_total
```

Comment (clwluvw) [reply README.md:36]: @SuperQ Good idea. Thanks. You are always welcome to submit PRs if you want. Anyway if you don't I'll submit another PR for this change.

**Comment by clwluvw** [comment]
ping @mehdy :D

**Comment by mehdy** [comment]
@clwluvw Would you please make the changes that were suggested in the review?

**Comment by clwluvw** [comment]
> @clwluvw Would you please make the changes that were suggested in the review?

I thought it would better to submit another PR because of PR title. But if you think it's okay i'll submit another commit on this PR?

**Comment by mehdy** [comment]
@clwluvw You could update the PR title though :) I think it'd be better if it's done with this PR.

**Comment by clwluvw** [comment]
@mehdy Done. please recheck if I miss something. Thanks!

**Pull request merged**